tests/core/Resolver.spec.ts

```diff
@@ -89,7 +89,7 @@ describe('Resolver', () => {
       const published = true;
       const didState = await resolver.resolve(didUniqueSuffix) as DidState;
       const resultingDocument = DocumentComposer.transformToExternalDocument(didState, `did:sidetree:${didUniqueSuffix}`, published);
-      fixtureDriftHelper(resultingDocument, afterUpdate, 'resolution/afterUpdate.json', OVERWRITE_FIXTURES);
+      fixtureDriftHelper(resultingDocument, afterUpdate, 'resolution/afterUpdate.json', false);
```

Comment:
Sanity: did you intend to make these false, if so, why?

Reply:
Good catch. Not intended.